### DISCUSSION TITLE
Adding v4l-utils in the core-image-sato

### DISCUSSION THREAD

**Discussion by VishwanathB45** [OE4T/meta-tegra#1351] [Q&A] [answered]
Hi,
I'm using kirkstone-l4t-r32.7.x branch in meta-tegra.I wanted to add the v4l-utils in the image sato.
I tried to add it using IMAGE_INSTALL:append = " v4l-utils" in the local.conf file.but it didn't work.Could you kindly share any insights or tips on how to accomplish this?

**Comment by madisongh** [comment]
Can you be a bit more specific than "it didn't work"?   Did it fail to build, or did the package not get added to the image, or something else?

**Reply by VishwanathB45** [reply]
Hi @madisongh 
Error was:
Nothing RPROVIDES 'v4l-utils' 
Runtime target 'v4l-utils' is unbuildable, removing...
Missing or unbuildable dependency chain was: ['v4l-utils']

**Reply by dwalkes** [reply, marked as answer]
I'm guessing something is messed up with your bitbake layers?  It should be in [meta-oe](https://layers.openembedded.org/layerindex/branch/master/recipes/?q=v4l-utils).  It works for me when I use the latest branch of tegra-demo-distro at https://github.com/OE4T/tegra-demo-distro/tree/kirkstone-l4t-r32.7.x so you might compare your setup to the setup used there.
